aggregator/aggregator.go

```diff
@@ -1310,6 +1314,24 @@ func (a *Aggregator) handleMonitoredTxResult(result ethtxmanager.MonitoredTxResu
 	}
 }
 
+func (a *Aggregator) waitForSyncronizerToSyncUp(ctx context.Context, batchNum *uint64) error {
```

Comment:
Nitpick, I think it is pronounced "synchronizer", with an H?

```suggestion
func (a *Aggregator) waitForSynchronizerToSyncUp(ctx context.Context, batchNum *uint64) error {
```

Reply:
Fix: https://github.com/0xPolygon/cdk-validium-node/pull/133/commits/8f20b1c15185f07d9a5778fc747aecdc89378e60